db/migrate/20201215114356_move_proposals_fields_to_i18n.decidim_proposals.rb

```diff
@@ -28,10 +28,7 @@ def up
           locale => proposal.body
         }
 
-        # rubocop:disable Rails/SkipsModelValidations
-        proposal.update_column("new_title", proposal.new_title)
-        proposal.update_column("new_body", proposal.new_body)
-        # rubocop:enable Rails/SkipsModelValidations
+        proposal.save(validate: false)
```

Comment:
Could you explain want you wanted to achieve here?

Reply:
Ah yes ! On the target branch this is the decidim-app v0.26 migrations directory. So in this PR I come back to the original migrations of the `master` branch

Reply:
Following [this change request](https://github.com/OpenSourcePolitics/decidim-nyc/pull/18#issuecomment-1278683422)